.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+  - "2.7"
+  - "3.3"
+  - "3.4"
```

Comment:
Not bad per se - just getting closer to completely obsolete. We can support it if you want, but I don't think we need to.